terraform/modules/tf_lambda/variables.tf

```diff
@@ -41,6 +41,11 @@ variable "source_object_key" {
   description = "S3 object key pointing to the function source code"
 }
 
+variable "concurrency_limit" {
+  default     = ""
```

Comment:
What does Lambda default to?

Reply:
It defaults to disabled, which is no value at all. However, Terraform [currently interprets 0 as disabled](https://github.com/terraform-providers/terraform-provider-aws/issues/3803), which is technically a bug.

I updated the default to `0` and added a comment and more complete description